plugin/path_creds.go

```diff
@@ -99,14 +95,14 @@ func (b *backend) credReadOperation(ctx context.Context, req *logical.Request, f
 
 	case role.LastVaultRotation == unset:
 		b.Logger().Debug("rotating password for the first time so Vault will know it")
-		resp, respErr = b.generateAndReturnCreds(ctx, req.Storage, roleName, role, cred)
+		resp, respErr = b.generateAndReturnCreds(ctx, engineConf, req.Storage, roleName, role, cred)
 
-	case role.PasswordLastSet.After(role.LastVaultRotation.Add(passwordLastSetBuffer)):
+	case role.PasswordLastSet.After(role.LastVaultRotation.Add(time.Second * time.Duration(engineConf.PasswordLastSetBuffer))):
 		b.Logger().Debug(fmt.Sprintf(
```

Comment:
I think we should raise this to Info(), maybe even Warning(). Seems like if you're hitting this case then you need to adjust the buffer or else your passwords are are getting rotated more often than you configured/expect, or something else is changing the password unexpectedly which is definitely of interest. Either way you'd want this surfaced, and not just a debug message.

Reply:
Yes, good point! Will raise it to the warn level because yeah, this is bad and someone could end up debugging it.